robottelo/ui/user.py

```diff
@@ -109,13 +115,11 @@ def delete(self, name, really=True):
             locators['users.delete'],
         )
 
-    def update(self, username, new_username=None,
-               email=None, password=None,
-               first_name=None, last_name=None, locale=None,
-               roles=None, new_roles=None, locations=None,
-               new_locations=None, organizations=None,
-               new_organizations=None, default_org=None,
-               default_loc=None, select=False):
+    def update(self, username, new_username=None, email=None, password=None,
+               first_name=None, last_name=None, locale=None, roles=None,
+               new_roles=None, locations=None, new_locations=None,
+               organizations=None, new_organizations=None, default_org=None,
+               default_loc=None, select=False, submit=True):
```

Comment:
Hmm. Can't agree with you as I like to check negative scenario with `submit=False`. We definitely can have a defect with that functionality, when, for example, we click `Cancel` button, but changes submitted anyway. Same for `create` procedure